### PR TITLE
Fix a bug in pip install

### PR DESCRIPTION
**Pull request by zheng-da**
*Issue #, if available:*

*Description of changes:*
We should install `scikit-learn` instead of `sklearn`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def find_version(*file_paths):
     'transformers>=4.3.0',
     'pandas',
     'pylint',
-    'sklearn',
+    'scikit-learn',
     'ogb',
```
Comment (thvasilo) [inline setup.py:38]: Should have a `>=` for the ogb version here as well since we had problems before?

Comment (classicsong) [reply setup.py:38]: Can we fix it with ogb == 1.3.6?
Furthermore, we would better add torch >= 1.12

Comment (thvasilo) [reply setup.py:38]: Yeah if it's going to be more than one change let's open another issue and PR to keep this one focused.

Comment (thvasilo) [reply setup.py:38]: Opened #90 , if you can think of more _minimum_ requirements let's add them. I would suggest against pinning specific versions unless it's absolutely necessary as that would make installation for end users more difficult. "Larger or equal" tends to be good enough.

We can pin dependencies in our Dockerfiles to ensure working environment.

**Pull request merged**